File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.cs
using System;

public class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        string arr = "";
        int case1 = int.Parse(a.ToString() + b.ToString());
        int case2 = int.Parse(b.ToString() + a.ToString());
        answer = case1 > case2 ? case1 : case2;
        return answer;
    }
}

File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int a, int b) {
    int answer = 0;
    string str1 = to_string(a);
    string str2 = to_string(b);
    answer = stoi(str1 + str2) > stoi(str2 + str1) ? stoi(str1 + str2) :  stoi(str2 + str1);
    return answer;
}